coderd/notifications/enqueuer.go

```diff
@@ -81,6 +89,7 @@ func (s *StoreEnqueuer) Enqueue(ctx context.Context, userID, templateID uuid.UUI
 		Payload:                input,
 		Targets:                targets,
 		CreatedBy:              createdBy,
+		CreatedAt:              s.clock.Now().UTC(), // mimicking dbtime.Now()
```

Comment:
`dbtime` rounds to the nearest microsecond, so "mimicking" is a bit confusing here.

Should we make `dbtime.Now` take an optional quartz argument `func Now(c ...quartz.Clock)`? 😄